src/data/polynomial/eval.lean

```diff
@@ -320,6 +320,21 @@ begin
     simp only [mul_assoc, C_mul_monomial, eval_monomial], }
 end
 
+lemma eval_monomial_one_add_sub [comm_ring S] (d : ℕ) (y : S) :
+  eval (1 + y) ((monomial d) (d + 1 : S)) - eval y ((monomial d) (d + 1 : S)) =
```

Comment:
```suggestion
  eval (1 + y) (monomial d (d + 1 : S)) - eval y (monomial d (d + 1 : S)) =
```
Can you maybe also add a doc string to this lemma that expresses the formula in math?

Reply:
Done, thank you!